File: DivideFraction.js
numerators = [16, 9, 640, 7];
denominators = [9, 20, 27, 20];

function doDivide() {
    var temp;
    // Invert every other fraction but the first
    for (var i = 1; i < numerators.length; i++) {
        temp = numerators[i];
        numerators[i] = denominators[i];
        denominators[i] = temp;
    }
    doMultiply();
}

File: DivideFraction.cs
using System.Collections.Generic;

namespace Algebra_CS
{
    class DivideFraction : MultiplyFraction
    {
        public DivideFraction(List<int> num, List<int> denom) : base(num, denom)
        {
        }

        public int[] doDivide()
        {
            int temp;
            // Invert every other fraction but the first
            for (int i = 1; i < numerators.Count; i++)
            {
                temp = numerators[i];
                numerators[i] = denominators[i];
                denominators[i] = temp;
            }
            return doMultiply();
        }
    }
}
